packages/server/src/api/bannerRouter.ts

```diff
@@ -54,6 +55,45 @@ interface PuzzlesDataResponse {
     debug?: Debug;
 }
 
+const addLocalLanguageContent = (
+    test: BannerTest,
+    variant: BannerVariant,
+    country: string,
+    moduleName: string,
+): { bannerContent?: BannerContent; mobileBannerContent?: BannerContent } => {
+    const { bannerContent, mobileBannerContent } = variant;
+
+    if (moduleName === 'EuropeMomentLocalLanguageBanner') {
+        const localLanguage = countryCodeToLocalLanguageBannerHeader(
+            test.name,
+            variant.name,
+            country,
+            { bannerHeader: variant.bannerContent?.heading },
```

Comment:
actually this is still not quite right as we're passing in `bannerContent?.heading` as the default, but for mobile we want `mobileBannerContent?.heading`
I'll have another go...